node_frequency: 10
start_num: 0

topics:
  subscribers:
    pause: '/pause'
  publishers:
    count: '/count'
